### PR TITLE
 Adding bower.json

### PR DESCRIPTION
**Pull request by karan-kang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+*This fork will no longer be maintained once the original library has it's own bower.json.*
```
Comment (pioug) [inline README.md:1]: Can you undo this change please 🙏

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "keywords": [
+    "md-virtual-repeater",
+    "angular/material",
+    "angular"
+  ],
+  "authors": [
+    "Gilles Piou <gilles.piou@gmail.com>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/Karankang007/md-virtual-repeater"
```
Comment (pioug) [inline bower.json]: Maybe you can redirect to my repository `https://github.com/pioug/md-virtual-repeater` if you are not planning to maintain your fork

**Review by pioug — CHANGES_REQUESTED** [review]

**Comment by pioug** [comment]
Hey! I am glad there are still people using Bower 😁  I added few comments to your PR, just details. I will register the package on Bower after merging!

Thanks for your help!

File: README.md (hunk 1 above)
Comment (karan-kang) [reply README.md:1]: Hey, i will redirect the bower repo, once your git contains bower.json file.

**Review by pioug — APPROVED** [review]

**Comment by pioug** [comment]
@Karankang007 You can update the bower registry now 👍

**Comment by karan-kang** [comment]
Done. "md-virtual-repeater" bower package now points to your repo.

Please update your project ReadMe.md accordingly.

**Pull request merged**